Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/SetPoints/ReturnFanDirectPressure.mo

```diff
@@ -77,6 +77,8 @@ block ReturnFanDirectPressure
   Buildings.Controls.OBC.CDL.Logical.Switch swi
     "Return fan discharge static pressure setpoint"
     annotation (Placement(transformation(extent={{80,-100},{100,-80}})));
+  Buildings.Controls.OBC.CDL.Continuous.Division div "Normalized the control error"
```

Comment:
This implementation needs to be changed. Currently the controller takes at `u_m` the setpoint and at `u_s` the measured values, which is just the opposite of what it should be.  Also, please let the control error be computed inside `conP` and not outside.